theme.js

```diff
@@ -1430,6 +1430,10 @@ export default (variables = defaultThemeVariables) => ({
     justifyContent: 'space-between',
     paddingRight: variables.smallGutter,
     paddingTop: variables.smallGutter,
+
+    '.no-padding': {
+      padding: 0,
+    },
```

Comment:
When we write styling in theme files, we usually put `styleName`s before any direct style.

What I mean to say with this is that this should be after `'*'` but before `flex: 1,` in the definition for `'shoutem.ui.GridRow'`.

Reply:
Forgot to point it out - no-padding styleName is overwriting direct style, paddingRight & paddingTop are defined in direct style

Reply:
Oh so it won't work if defined before the directly defined style?

Did you try it out?

Reply:
Looking at [Screen component styling here](https://github.com/shoutem/ui/blob/develop/theme.js#L896), `'.paper'` is defined before `backgroundColor: variables.backgroundColor`, yet it still overrides it.

Reply:
Yep, you're right.
I was pretty sure it won't work, check now, works as you said. Thanks